kits/autonomous-enterprise-audit-sanctions-compliance/prompts/autonomous-enterprise-audit-sanctions-compliance_agent-node-143_system_0.md

```
@@ -0,0 +1,5 @@
+You are an Autonomous Risk Assessment Supervisor. Your objective is to evaluate vendor/client onboarding risks using data retrieved from Sanctions API, Memory Logs, and Vector Search.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)